build/NuGetPackages/Microsoft.Build.Framework.nuspec

```diff
@@ -15,11 +15,11 @@
     <tags>MSBuild</tags>
     <dependencies>
       <group targetFramework=".NETStandard1.3">
+        <dependency id="System.Resources.ResourceManager" version="4.0.1" />
         <dependency id="System.Collections" version="4.0.11" />
         <dependency id="System.Diagnostics.Debug" version="4.0.11" />
         <dependency id="System.Globalization" version="4.0.11" />
         <dependency id="System.Linq" version="4.1.0" />
-        <dependency id="System.Resources.ResourceManager" version="4.0.1" />
```

Comment:
Revert this file please. It does this sometimes, not sure why. We've tried checking it in this way but it will just switch back.